diff --git a/frontend/src/Store/Actions/seriesIndexActions.js b/frontend/src/Store/Actions/seriesIndexActions.js
index 65c20870f8..dc4566b3a2 100644
--- a/frontend/src/Store/Actions/seriesIndexActions.js
+++ b/frontend/src/Store/Actions/seriesIndexActions.js
@@ -19,8 +19,8 @@ export const section = 'seriesIndex';
 export const defaultState = {
   sortKey: 'sortTitle',
   sortDirection: sortDirections.ASCENDING,
-  secondarySortKey: 'sortTitle',
-  secondarySortDirection: sortDirections.ASCENDING,
+  secondarySortKey: 'added',
+  secondarySortDirection: sortDirections.DESCENDING,
   view: 'posters',
 
   posterOptions: {
