diff --git a/frontend/src/Series/useSeries.ts b/frontend/src/Series/useSeries.ts
index 63497121dd..f1a1e567b4 100644
--- a/frontend/src/Series/useSeries.ts
+++ b/frontend/src/Series/useSeries.ts
@@ -589,8 +589,8 @@ export const useSeriesIndex = () => {
       customFilters,
       sortKey: sortKey as keyof Series,
       sortDirection,
-      secondarySortKey: 'sortTitle',
-      secondarySortDirection: 'ascending',
+      secondarySortKey: 'added',
+      secondarySortDirection: 'descending',
       sortPredicates: SORT_PREDICATES,
     });
   }, [customFilters, seriesData, selectedFilterKey, sortKey, sortDirection]);
