diff --git a/frontend/src/Components/Page/Header/PageHeader.js b/frontend/src/Components/Page/Header/PageHeader.js
index 7e06ee1c31..4af62d9a51 100644
--- a/frontend/src/Components/Page/Header/PageHeader.js
+++ b/frontend/src/Components/Page/Header/PageHeader.js
@@ -85,14 +85,6 @@ class PageHeader extends Component {
               null
           }
 
-          <IconButton
-            className={styles.donate}
-            name={icons.HEART}
-            aria-label={translate('Donate')}
-            to="https://sonarr.tv/donate.html"
-            size={14}
-            title={translate('Donate')}
-          />
           <PageHeaderActionsMenuConnector
             onKeyboardShortcutsPress={this.onOpenKeyboardShortcutsModal}
           />
diff --git a/frontend/src/System/Status/Status.tsx b/frontend/src/System/Status/Status.tsx
index ae1636b3e0..0e0c6a8d4a 100644
--- a/frontend/src/System/Status/Status.tsx
+++ b/frontend/src/System/Status/Status.tsx
@@ -5,7 +5,6 @@ import translate from 'Utilities/String/translate';
 import About from './About/About';
 import DiskSpace from './DiskSpace/DiskSpace';
 import Health from './Health/Health';
-import MoreInfo from './MoreInfo/MoreInfo';
 
 class Status extends Component {
   //
@@ -18,7 +17,6 @@ class Status extends Component {
           <Health />
           <DiskSpace />
           <About />
-          <MoreInfo />
         </PageContentBody>
       </PageContent>
     );
